DSharpPlus.HttpInteraction.AspNetCore/DSharpPlus.HttpInteraction.AspNetCore.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
can you explain why it's set up like this? wouldn't it make sense to make this the SDK?

Reply:
I looked at this article: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/target-aspnetcore?view=aspnetcore-7.0&tabs=visual-studio#use-the-aspnet-core-shared-framework

Reply:
> As of .NET Core 3.0, projects using the Microsoft.NET.Sdk.Web MSBuild SDK implicitly reference the shared framework.

Reply:
so, `<Project Sdk="Microsoft.NET.Sdk.Web>` as first line?